docs/schema/authenticationdata-element-sps15xsdsearchset2.md

```diff
@@ -7,14 +7,14 @@ ms.topic: article
 ms.prod: sharepoint
 ms.localizationpriority: medium
 ms.assetid: 8ca8cbf0-9123-8716-4de7-d13014e06f7a
-description: "Last modified: March 09, 2015"
+description: AuthenticationData element (SPS15XSDSearchSet2) is an XML element.
```

Comment:
For these articles that don't have much information in the intro paragraph, you can also use this format instead: "Learn about the AuthenticationData element (SPS15XSDSearchSet2)."